physics/mynnedmf_wrapper.F90

```diff
@@ -25,25 +25,25 @@ subroutine mynnedmf_wrapper_init (          &
 
         implicit none
         
-        logical,             intent(in)  :: do_mynnedmf
-        logical,             intent(in)  :: lheatstrg
-        character(len=*),    intent(out) :: errmsg
-        integer,             intent(out) :: errflg
-
-        real(kind=kind_phys),intent(in)  :: con_xlv
-        real(kind=kind_phys),intent(in)  :: con_xlf
-        real(kind=kind_phys),intent(in)  :: con_rv
-        real(kind=kind_phys),intent(in)  :: con_rd
-        real(kind=kind_phys),intent(in)  :: con_ep2
-        real(kind=kind_phys),intent(in)  :: con_grav
-        real(kind=kind_phys),intent(in)  :: con_cp
-        real(kind=kind_phys),intent(in)  :: con_cpv
-        real(kind=kind_phys),intent(in)  :: con_rcp
-        real(kind=kind_phys),intent(in)  :: con_p608
-        real(kind=kind_phys),intent(in)  :: con_cliq
-        real(kind=kind_phys),intent(in)  :: con_cice
-        real(kind=kind_phys),intent(in)  :: con_karman
-        real(kind=kind_phys),intent(in)  :: con_t0c
+        logical,        intent(in)  :: do_mynnedmf
+        logical,        intent(in)  :: lheatstrg
```

Comment:
Since the check for `lheatstrg` has been removed and it isn't used elsewhere, we can remove it from the argument list and metadata.

Reply:
Good catch. This lheatstrg needs to go.